### ISSUE TITLE
Replace popper.js with native support 

### ISSUE THREAD

**Issue by blackfalcon** [AlaskaAirlines/auro-popover#44]
> @Totati and @mfreed7 thanks a lot for this issue and for alerting us of the problem. Sorry, were unable to release an update prior to the 114 release of Chrome, but when we discovered the problem, we were presently surprised to find this issue with the fix outlined.
> 
> Thanks again! OSS FOR THE WIN!

Great news! Thanks for the update, and for updating Auro so quickly.

You might also look into the [new Popover API](https://developer.chrome.com/blog/introducing-popover-api/) as a replacement for the popover functionality in Auro. I'd be curious to hear your feedback about the feature and see if it can reduce the JS required to implement popovers.

_Originally posted by @mfreed7 in https://github.com/AlaskaAirlines/auro-popover/issues/42#issuecomment-1572693084_

### Additional context

The new API is out in the latest release of Chrome, but we will require additional browser support before considering this feature. 

https://caniuse.com/?search=popover

**Comment by mfreed7** [comment]
> The new API is out in the latest release of Chrome, but we will require additional browser support before considering this feature.
> 
> https://caniuse.com/?search=popover

I agree browser support is still in progress. However, both WebKit and Gecko are supportive and working on implementations. So perhaps soon!

One other thing to point out: popper.js does two things really. One is to render the popover on top of everything else on the page, without getting clipped or transformed. That's what the Popover API does. The other thing popper.js does is help you *position* the popover. For that, there's another upcoming feature, [CSS Anchor Positioning](https://drafts4.csswg.org/css-anchor-position-1/). That one is a bit behind the Popover API, but not by too much.
